src/commentsDashboard/index.jsx

```diff
@@ -73,26 +52,24 @@ const CommentsDashboard = () => {
                 key={user.id}
                 variant={activeUser === user.id ? "contained" : "outlined"}
                 color={activeUser === user.id ? "success" : ""}
-                onClick={() => newActiveUser(user.id)}
+                onClick={() => setNewActiveUser(user.id)}
               >
                 {user.username}
               </Button>
             ))}
           </ButtonGroup>
-          {activeUser && displayMode === DisplayMode.posts && (
-            <div>
-              <PostsList activeUser={activeUser} />
-            </div>
-          )}
-          {activeUser && displayMode === DisplayMode.photos && (
-            <div>
-              <PhotosList activeUser={activeUser} />
-            </div>
-          )}
+          {activeUser &&
+            (displayMode === DisplayMode.posts ? (
+              <div>
+                <PostsList activeUser={activeUser} />
+              </div>
+            ) : (
+              <div>
+                <PhotosList activeUser={activeUser} />
+              </div>
+            ))}
```

Comment:
You are rendering the dix no matter if the condition is met. So you can either wrap the whole conditional expression with the div, or get rid of the div if you don't need it.

Reply:
done